lib/board.rb

```diff
@@ -26,4 +26,44 @@ def initialize
   def valid_coordinate?(cell)
     @cells.has_key?(cell)
   end
+
+  def valid_placement?(boat, coordinates)
+    if coordinates.count == boat.length
+          row = coordinates.map do |coordinate|
+            coordinate[0]
+          end
+
+          num_row = row.map do |letter|
+            letter.codepoints
+          end.flatten
```

Comment:
I don't love calling code point I feel like there has to be a better way to make sure that the letters are sequential. I also don't like calling .flatten on end. If you have ideas on how else to do this, let me know.

Reply:
Decided to use .ord instead of .codepoints